pkg/ui/workspaces/cluster-ui/src/graphs/bargraph/plugins.ts

```diff
@@ -73,6 +73,13 @@ const generateSeriesLegend = (uPlot: uPlot, seriesLegend: HTMLDivElement) => {
   });
 };
 
+// if the value has no decimal value, returns as is,
+// otherwise fixes to 2 decimals.
+function getFormattedValue(value: number): string {
+  if (value % 1 == 0) return String(value);
```

Comment:
TIL the `%` operator works with non-integer values. 🤯 (So it's more "remainder" than it is "modular arithmetic.")  There is also [`Number.isInteger()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger) in case you like it better?